cabal-testsuite/PackageTests/NewSdist/DeterministicTrivial/deterministic.test.hs

```diff
@@ -21,4 +21,5 @@ main = cabalTest $ do
     known <- liftIO (BS.readFile knownSdist)
     unknown <- liftIO (BS.readFile mySdist)
 
-    assertEqual "hashes didn't match for sdist" (BS16.encode $ SHA256.hash known) (BS16.encode $ SHA256.hash unknown)
+    skipIf "#8356" True $
```

Comment:
could you add a comment that explains that we don't use exitFailure because it succeeds locally but fails in ci?

Reply:
sure

Reply:
And you don't need to remove the `assertEqual`. Just put `skip` first and it skips before the dwarves that calculate all these things get to the `assertEqual`.

Reply:
(and no dollar after `skipIf`)